perf/yb-connect.yaml

```diff
@@ -4,10 +4,29 @@ services:
     container_name: kafka-connect
     ports:
       - 8083:8083
+      - 9875:9875
+      - 9876:9876
     environment:
       BOOTSTRAP_SERVERS: ${KAFKA_HOST}
       GROUP_ID: 1
       CONFIG_STORAGE_TOPIC: my-connect-configs
       OFFSET_STORAGE_TOPIC: my-connect-offsets 
       HOST_NAME: "0.0.0.0"
       ADVERTISED_HOST_NAME: ${DOCKER_IP}
+      KAFKA_JMX_PORT: '9875'
+      KAFKA_OPTS: "-javaagent:/usr/local/bin/jmx_prometheus_javaagent-0.9.jar=9876:/etc/jmx-exporter/kafka-connect.yml"
+    volumes:
+      - ./jmx-exporter/kafka-connect.yml:/etc/jmx-exporter/kafka-connect.yml
+      - ./jmx-exporter/jmx_prometheus_javaagent-0.9.jar:/usr/local/bin/jmx_prometheus_javaagent-0.9.jar
+
+  prometheus:
+    ports:
+      - 9090:9090
+    image: prom/prometheus
+    volumes:
+      - ./kafka-connect-prometheus.yml:/etc/prometheus/prometheus.yml
+
+  grafana:
```

Comment:
Grafana dashboard should be setup automatically. Check how to do that in [Debezium example] (https://github.com/debezium/debezium-examples/tree/main/monitoring/debezium-grafana)

It contains files to setup a datasource and multiple dashboard yaml files. You can start with one of the dashboard files but it will have to be changed for metrics specific to Yugabyte.